idaes/models/flowsheets/tests/test_methanol_flowsheet.py

```diff
@@ -0,0 +1,673 @@
+            for t, c in unit.pressure_balance.items():
+                assert pytest.approx(1e-5, rel=1e-5) == \
+                    iscale.get_constraint_transform_applied_scaling_factor(c)
+        if hasattr(unit, 'sum_of_phase_fractions'):
+            for t, c in unit.sum_of_phase_fractions.items():
+                assert pytest.approx(1e2, rel=1e-5) == \
+                    iscale.get_constraint_transform_applied_scaling_factor(c)
+        if hasattr(unit, "material_accumulation_disc_eq"):
+            for (t, p, j), c in unit.material_accumulation_disc_eq.items():
+                assert pytest.approx(1, rel=1e-5) == \
+                    iscale.get_constraint_transform_applied_scaling_factor(c)
+
+        if hasattr(unit, "energy_accumulation_disc_eq"):
+            for (t, p), c in unit.energy_accumulation_disc_eq.items():
+                assert pytest.approx(1e-3, rel=1e-5) == \
+                    iscale.get_constraint_transform_applied_scaling_factor(c)
+
+        if hasattr(unit, "element_accumulation_disc_eq"):
+            for (t, e), c in unit.element_accumulation_disc_eq.items():
+                assert pytest.approx(1, rel=1e-5) == \
+                    iscale.get_constraint_transform_applied_scaling_factor(c)
+
+    # equality constraints between ports at Arc sources and destinations
+    for arc in model.fs.component_data_objects(Arc, descend_into=True):
+        for c in arc.component_data_objects(Constraint, descend_into=True):
+            if hasattr(unit, "enth_mol_equality"):
+                for t, c in unit.enth_mol_equality.items():
+                    assert pytest.approx(1e-3, rel=1e-5) == \
+                        (iscale.
+                         get_constraint_transform_applied_scaling_factor(c))
+            if hasattr(unit, "flow_mol_equality"):
+                for t, c in unit.flow_mol_equality.items():
+                    assert pytest.approx(1, rel=1e-5) == \
+                        (iscale.
+                         get_constraint_transform_applied_scaling_factor(c))
+            if hasattr(unit, "mole_frac_comp_equality"):
+                for (t, j), c in unit.mole_frac_comp_equality.items():
+                    assert pytest.approx(1e2, rel=1e-5) == \
+                        (iscale.
+                         get_constraint_transform_applied_scaling_factor(c))
+            if hasattr(unit, "pressure_equality"):
+                for t, c in unit.pressure_equality.items():
+                    assert pytest.approx(1e-5, rel=1e-5) == \
+                        (iscale.
+                         get_constraint_transform_applied_scaling_factor(c))
+
+
+@pytest.mark.component
+def test_initialize_flowsheet(model):
+    solver = get_solver()
+    optarg = {'tol': 1e-6,
+              'max_iter': 5000}
+    solver.options = optarg
+    initialize_flowsheet(model)
+
+    assert degrees_of_freedom(model) == 0
+
+    assert model.fs.M101.outlet.flow_mol[0].value == pytest.approx(954, 1e-3)
```

Comment:
This is a rather high tolerance for checking the result (0.1%) - we should be able to check to a far higher tolerance that this (I would aim for `rel=1e-5`). It is probably also best to explicitly set this to a relative tolerance as well (i.e. do `rel=`) in case something changes in `pytest` in the future.